ui/job-view/headerbars/WatchedRepo.jsx

```diff
@@ -199,7 +199,7 @@ export default class WatchedRepo extends React.Component {
           )}
           <li className="watched-repo-dropdown-item">
             <a
-              href={`https://mozilla-releng.net/treestatus/show/${treeStatusName}`}
+              href={`https://mozilla-releng.net/static/ui/treestatus/show/${treeStatusName}`}
```

Comment:
Based on the comment from the bug, the domain should be `treestatus.mozilla-releng.net`.

Reply:
I have pushed the changes as requested . Please have a look @camd 👍 .